test/graph/identity_role_wrapper.go

```diff
@@ -80,3 +82,7 @@ func newIdentityRoleWrapper(g *TestGraph, params []interface{}) interface{} {
 func (w *identityRoleWrapper) IdentityRole() *rolePkg.IdentityRole {
 	return w.identityRole
 }
+
+func (w *identityRoleWrapper) Delete() {
+	w.graph.app.IdentityRoleRepository().Delete(context.Background(), w.identityRole.IdentityRoleID)
```

Comment:
The graph has a `context` object you can use, via `w.graph.ctx`.

Reply:
ok, I'll change that. Thanks!

Reply:
addressed in https://github.com/fabric8-services/fabric8-auth/pull/663/commits/9532179f8f23a606f0c99e42c5451f13a9f6a87c